Source/Core/Core/FreeLookConfig.cpp

```diff
@@ -37,7 +39,7 @@ void Config::Refresh()
 {
   if (!s_has_registered_callback)
   {
-    ::Config::AddConfigChangedCallback([] { Core::RunAsCPUThread([] { s_config.Refresh(); }); });
+    CPUThreadConfigCallback::AddConfigChangedCallback([] { s_config.Refresh(); });
```

Comment:
This likely needs the same complexity involved as in the `VideoConfig` as that is what the `FreeLookConfig` was patterned after.  FreeLook is this weird blend of inputs (the controller) and graphics (the "camera") and the config is used by both systems.

Reply:
The reason why it was added for VideoConfig was to stop the GPU thread from reading the config being VerifyValidity ran. There's nothing like VerifyValidity here, so we don't have the same problem.

I suppose that if we are to standards lawyer this, this is technically unsafe because there could be tearing, as there's no synchronization between the CPU thread writing and the GPU thread reading. But I think this would be better addressed by something simpler like a mutex, and this is far from the only config callback in the code base with this problem.